src/components/main.tsx

```diff
@@ -68,6 +99,9 @@ const Main = () => {
                             <img src={MetaMask} alt="Connect with Metamask" />
                         </Button>
                     </InputGroupAddon>
+                    <FormFeedback id="invalidAddress">
+                      Uh oh! Looks like this Wallet Address is Invalid!
```

Comment:
Minor nitpick, nouns in English in the middle of a sentence should not be capitalized, so it should read: "Looks like this wallet address is invalid"
And without the exclamation marks please (!), no need to yell at users